### PR TITLE
 compatible with ESX and QBCore frameworks

### PR DESCRIPTION
**Pull request by reyyghi**
added a bridge to work with ESX or QBCore frameworks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ Ultra customizable hud featuring a unique and robust settings menu. Change eveyt
 
 ![Project Sloth Buttons 2](https://user-images.githubusercontent.com/91661118/170895968-c4224105-5c87-4947-af94-8e8bd792f566.png)
 
+### Unsupported ESX Version made by [reyyghi](https://github.com/reyyghi)
```
Comment (MonkeyWhisper) [inline README.md:8]: Please only have a PR with these changes, the changes made to the README, everything else we will not merge into main branch.

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]
Please only have a PR with these changes, the changes made to the README, everything else we will not merge into main branch.

**Comment by MonkeyWhisper** [comment]
Hi @reyyghi 

First, thank you very much for this PR. I appreciate it very much! 

However, please note that our focus has always been QBCore and many of us have not had experience working with ESX therefore it will make things difficult whenever we do decide to do an update to also maintain the ESX version. 

Please only push the read me changes that show you're the one maintaining this ESX version and to link your GitHub.

File: README.md (hunk 1 above)
Comment (reyyghi) [reply README.md:8]: Okay, I understand

**Pull request closed without merging**